person.rb

```diff
@@ -13,6 +13,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
   end
```

Comment:
- [ ] From the project requirements, you are expected to implement the associations and you have done it which is good. Now, please note that `person` object has many rentals. In addition to what you have done, I would suggest you implement the `add_rental` method which look like this 👇 
```
 def add_rental(book, date)
   Rental.new(date, book, self)
 end
```